push-notifications/README.md

```diff
@@ -322,7 +324,7 @@ requestPermissions() => Promise<PermissionStatus>
 
 Request permission to receive push notifications.
 
-On Android it doesn't prompt for permission because you can always
+On Android 12 and below it doesn't prompt for permission because you can always
```

Comment:
this texts comes from the types, should be edited there and then run `npm run build` to propagate it to the README
https://github.com/ionic-team/capacitor-plugins/blob/main/push-notifications/src/definitions.ts#L93-L117